source/includes/fact-filter-queries.rst

```diff
@@ -0,0 +1,10 @@
+Allows you to filter and limit the amount of entries displayed in 
+the :guilabel:`Query Converter` pane.
+
+1. Enter a text string to match against the object name and SQL 
+   syntax in the :guilabel:`Filter` text box or click the 
+   :icon-fa5:`ellipsis-v` icon and select from the following options:
+
+   - :guilabel:`Successful conversions`
+   - :guilabel:`Incomplete conversions`
+   - :guilabel:`Failed conversions`
```

Comment:
(lists)
You might be able to make this a two step process. Otherwise let's omit the itemized list.